CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 - [Compare to latest release][unreleased]
 
+## [1.3.5] - 2017-03-07
```

Comment:
there are links at the bottom of the document that need to be updated as well. See [this commit](https://github.com/SparkPost/python-sparkpost/commit/8808080ed4cfe17eadb0114eb5b7c12fb311e7a7).